src/test/pages/UserContextReconfirmationModal.test.tsx

```diff
@@ -36,7 +36,7 @@ describe("UserContextReconfirmationModal", () => {
       role: "STUDENT",
       modifyUser: (user) =>
         produce(user, (u) => {
-          u.registeredContextsLastConfirmed = (MOST_RECENT_AUGUST().valueOf() - 10000000) as unknown as Date;
+          u.registeredContextsLastConfirmed = MOST_RECENT_AUGUST();
```

Comment:
The subtraction was removed. Was this intentional?

Reply:
Oh yes, I intended to remove all of them as it's unnecessary. MOST_RECENT_AUGUST() returns a timestamp in ms now.